azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -417,21 +451,39 @@ def supported(self):
             return self._cgroups_supported
 
         def enabled(self):
-            return self._cgroups_enabled
+            return self._agent_cgroups_enabled | self._extensions_cgroups_enabled
```

Comment:
NIT: Use boolean `or` operator instead of the `bitwise or: |` for boolean commands for better readability

Reply:
addressed.